cmd/copy.go

```diff
@@ -255,6 +255,20 @@ func (raw rawCopyCmdArgs) cook() (cookedCopyCmdArgs, error) {
 		azcopyScanningLogger.CloseLog()
 	})
 
+	/* We support DFS by using blob end-point of the account. We replace dfs by blob in src and dst */
+	if src,dst := inferArgumentLocation(raw.src), inferArgumentLocation(raw.dst);
+				src == common.ELocation.BlobFS() || dst == common.ELocation.BlobFS() {
+		if src == common.ELocation.BlobFS() && dst != common.ELocation.Local() {
```

Comment:
minor: add a comment to explain that we are trying to phase out the usage of dfs endpoint in general, hence why we are using blob endpoint even when the source is dfs.